### ISSUE TITLE
Spelling mistakes in process.xhtml howto

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6407]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6407|
|Type|defect|
|Created|2013-04-06 00:52:51Z|

`doc/core/howto/process.xhtml` has a spelling mistake: "receieved".

Attachments:

* [fixed_spelling_error.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/437/43723f292d395ad92ccbf54a063c5aa7e6eb24dc/c33c26b408d3ebcb32488a82efbd0a4682b7a858.patch) (728 bytes) - added by stephsolis on 2013-04-17 21:19:51Z - Fixed the spelling error
* [fixed_spelling_errors_v2.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/437/43723f292d395ad92ccbf54a063c5aa7e6eb24dc/f5a46286f79455a6177f95c382d4ef0f08a7efa3.patch) (1941 bytes) - added by stephsolis on 2013-04-18 15:20:48Z - Fixed more spelling mistakes and added news file

<details><summary>Searchable metadata</summary>

```
trac-id__6407 6407
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation__easy documentation, easy
time__1365209571000000 1365209571000000
changetime__1474198049187875 1474198049187875
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_3"></a>|@wallrj set owner to @wallrj<br>@wallrj set status to `assigned`|
|-|-|

reviewing...

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_4"></a>|@wallrj set owner to stephsolis<br>@wallrj set status to `new`|
|-|-|

## Code Review

Thanks for the patch stephsolis.

 1. There are some other spelling mistakes in that document which you
    may as well fix.
    1. desireable
    2. parseable
    3. addtional
    4. ...check for others. (note American English spelling)
 2. Add a .doc news file.

-RichardW.

**Comment by twisted-trac** [comment]
|<img alt="stephsolis's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|stephsolis removed owner|
|-|-|

Thanks for the review rwall. I've applied your suggestions.

**Comment by twisted-trac** [comment]
|[<img alt="wallrj's avatar" src="https://avatars.githubusercontent.com/u/978965?s=50" width="50" height="50">](https://github.com/wallrj)<a name="note_6"></a>|@wallrj set status to `closed`|
|-|-|

(In [38158]) Apply fixed_spelling_errors_v2.patch: Fix spelling mistakes in the Using Processes howto

Author: stephsolis
Reviewer: rwall
Fixes: [#6407](https://github.com/twisted/twisted/issues/6407)

Fix spelling mistakes in the Using Processes howto.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
